system/web/context/ExceptionBean.cfc

```diff
@@ -35,7 +35,7 @@ component accessors="true"{
 		any extraMessage = "",
 		any extraInfo = ""
 	){
-		variables.exceptionStruct = arguments.errorStruct;
+		variables.exceptionStruct = duplicate(arguments.errorStruct);
```

Comment:
Can you explain specifically what this is doing.  This code actually used to duplicate the exception, but it causes issues because duplicate changes the underlying type of the object so we removed it.  Changing this back will likely cause regressions.  Here you can read one of the issues that duplicate caused people in the past:
https://groups.google.com/forum/#!topic/coldbox/yVZnyOyMOHU